apps/condo/domains/ticket/schema/IncidentExportTask.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Generated by `createschema ticket.IncidentExportTask 'status:Select:processing,completed,error; format:Select:excel; exportedRecordsCount:Integer; totalRecordsCount:Integer; file?:File; meta?:Json; where:Json; sortBy:Json; locale:Text; timeZone:Text; user:Relationship:User:CASCADE;'`
+ */
+
+const conf = require('@open-condo/config')
+const { canOnlyServerSideWithoutUserRequest } = require('@open-condo/keystone/access')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { extractReqLocale } = require('@open-condo/locales/extractReqLocale')
+
+const { EXPORT_STATUS_VALUES, PROCESSING, EXCEL, COMPLETED, ERROR } = require('@condo/domains/common/constants/export')
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { normalizeTimeZone } = require('@condo/domains/common/utils/timezone')
+const { DEFAULT_ORGANIZATION_TIMEZONE } = require('@condo/domains/organization/constants/common')
+const access = require('@condo/domains/ticket/access/IncidentExportTask')
+const { exportIncidents } = require('@condo/domains/ticket/tasks/exportIncidents')
+
+const { getFileMetaAfterChange } = FileAdapter
+
+
+const INCIDENT_EXPORT_TASK_FOLDER_NAME = 'IncidentExportTask'
+const IncidentExportTaskFileAdapter = new FileAdapter(INCIDENT_EXPORT_TASK_FOLDER_NAME)
+const setFileMetaAfterChange = getFileMetaAfterChange(IncidentExportTaskFileAdapter, 'file')
+
+const IncidentExportTask = new GQLListSchema('IncidentExportTask', {
+    schemaDoc: 'Stores requested export format, status of export job, link to exported file and information about progress of export job',
+    fields: {
+
+        status: {
+            schemaDoc: 'Status of the job. Used at client side progress information component',
+            type: 'Select',
+            options: EXPORT_STATUS_VALUES,
+            defaultValue: PROCESSING,
+            isRequired: true,
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
```

Comment:
What is the point of `canOnlyServerSideWithoutUserRequest`? Why not `create: false`?

Reply:
`canOnlyServerSideWithoutUserRequest` is function
```
const canOnlyServerSideWithoutUserRequest = () => false
```
if use simple `false` then you catch error like `Field "exportedRecordsCount" is not defined by type "IncidentExportTaskUpdateInput".`